integration-tests/api/__tests__/admin/user.js

```diff
@@ -354,3 +357,74 @@ describe("/admin/users", () => {
     })
   })
 })
+
+describe("[MEDUSA_FF_ANALYTICS] /admin/analytics-config", () => {
+  let medusaProcess
+  let dbConnection
+
+  beforeAll(async () => {
+    const cwd = path.resolve(path.join(__dirname, "..", ".."))
+    const [process, connection] = await startServerWithEnvironment({
+      cwd,
+      env: { MEDUSA_FF_ANALYTICS: true },
+    })
+    dbConnection = connection
+    medusaProcess = process
+  })
+
+  afterAll(async () => {
+    const db = useDb()
+    await db.shutdown()
+
+    medusaProcess.kill()
+  })
+
+  describe("DELETE /admin/users", () => {
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+      await userSeeder(dbConnection)
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("Deletes a user and their analytics config", async () => {
+      const api = useApi()
+
+      const userId = "member-user"
+
+      await simpleAnalyticsConfigFactory(dbConnection, {
+        user_id: userId,
+      })
+
+      const response = await api.delete(`/admin/users/${userId}`, {
+        headers: { Authorization: "Bearer test_token" },
```

Comment:
suggestion: as mention by phil, the headers can be moved to a const to be used accros the test cases